ledger/tracker.go

```diff
@@ -417,7 +454,7 @@ func (tr *trackerRegistry) commitSyncer(deferredCommits chan *deferredCommitCont
 }
 
 // commitRound commits the given deferredCommitContext via the trackers.
-func (tr *trackerRegistry) commitRound(dcc *deferredCommitContext) {
+func (tr *trackerRegistry) commitRound(dcc *deferredCommitContext) (err error) {
```

Comment:
I'd personally prefer to not use named return values here, or at least avoid the naked returns. Whenever I see a naked return I have to go back and consider the prototype to figure out what else is being returned (maybe it has to be zeroed).

Reply:
Agreed, I really dislike named returns for long functions